packages/solid-crs-manage/lib/features/object/object-root.component.ts

```diff
@@ -160,11 +170,37 @@ export class ObjectRootComponent extends RxLitElement {
       ${ this.state?.matches(ObjectStates.EDITING) ? html`<div slot="actions"><button class="no-padding inverse cancel" @click="${() => this.actor.send(ObjectEvents.CANCELLED_EDIT)}">${unsafeSVG(Cross)}</button></div>` : '' }
       <div slot="actions"><button class="no-padding inverse delete" @click="${() => this.actor.send(ObjectEvents.CLICKED_DELETE, { object: this.object })}">${unsafeSVG(Trash)}</button></div>
     </nde-content-header>
+    <div class="content-and-sidebar">
+
+    <nde-sidebar>
+      <nde-sidebar-item>
+        <div slot="content">
+          Een week geleden aangepast door Pieter Pieters
+        </div>
+      </nde-sidebar-item>
+      <nde-sidebar-item .padding="${false}" .showBorder="${false}">
+        <nde-sidebar-list slot="content">
+          <nde-sidebar-list-item slot="item" isTitle>
```

Comment:
```suggestion
          <nde-sidebar-list-item slot="title" isTitle>
```

To align with the primary sidebar's title

Reply:
`slot=title` does not allign with figma